pyanaconda/modules/storage/partitioning/schedule.py

```diff
@@ -57,10 +56,6 @@ def get_candidate_disks(storage):
         if not disk.format.supported or disk.protected:
             continue
 
-        if storage.config.clear_part_disks and \
-           (disk.name not in storage.config.clear_part_disks):
-            continue
```

Comment:
@vojtechtrefny, we collect the candidate disks for autopart and reqpart after the disks are cleared and initialized and all unselected devices are hidden, so it seems safe to drop this check. What do you think?

Reply:
Yes, it should be safe to remove this, if this functions after clearpart.